samples/websockets/route.yaml

```diff
@@ -30,4 +30,3 @@ spec:
     - destination:
         host: tornado
       weight: 100
-    websocketUpgrade: true
```

Comment:
Please explain why this seemingly unrelated change was included in this PR.

Reply:
@frankbu @geeknoid

Reply:
It's a while ago, so I can't remember. It looks like it might have been accidentally added to the wrong branch, but it does look correct to remove it, since that field is not documented or supported.